python/triton/compiler/compiler.py

```diff
@@ -73,7 +73,7 @@ def optimize_ttgir(mod, num_stages, arch):
     pm.add_tritongpu_remove_layout_conversions_pass()
     pm.add_tritongpu_optimize_dot_operands_pass()
     # TODO enable this pass for AMD GPU when it is ready
-    if torch.version.hip is None:
+    if is_hip():
```

Comment:
IIRC, the pipeline pass is disabled by Luxsoft. So the logic here should be something like
```
if not is_hip():
  pm.add_tritongpu_pipeline_pass(num_stages)
```

Or if the luxsoft's work is mature enough to enable the pipeline pass, we should simply remove the check hip here.